tests/test_pipenv.py

```diff
@@ -440,6 +440,26 @@ def test_windows_pinned_pipfile(self):
             assert 'tablib' in p.lockfile['default']
 
 
+    @pytest.mark.e
+    @pytest.mark.install
+    @pytest.mark.vcs
+    @pytest.mark.resolver
+    def test_editable_vcs_install_in_pipfile_with_dependency_resolution_doesnt_traceback(self):
+        # See https://github.com/pypa/pipenv/issues/1240
+        with PipenvInstance() as p:
+            with open(p.pipfile_path, 'w') as f:
+                contents = """
+[packages]
+requests = {git = "https://github.com/requests/requests.git", editable = true}
+"oslo.utils" = "==1.4.0"
+                """.strip()
+                f.write(contents)
+            c = p.pipenv('install')
+            assert c.return_code == 1
+            assert 'FileNotFoundError' not in c.out
+            assert 'FileNotFoundError' not in c.err
```

Comment:
Can we get a positive assertion here? Right now this could fail and produce a trace back for any reason except FileNotFoundError. That’s contradictory to what the test claims to be checking. What are we expecting to happen in this case?

Reply:
I've tightened up the assertions here, so they match the test name a bit better.